src/renderer/components/uielements/input/Input.style.tsx

```diff
@@ -16,7 +16,7 @@ const fontSettings: FontSettings = {
     spacing: '0.5px'
   },
   large: {
-    size: key('sizes.font.normal', '12px'),
+    size: '24px',
```

Comment:
set it manually as we dont have large enough values [at the palette](https://gitlab.com/thorchain/asgardex-common/asgardex-theme/-/blob/master/src/light.ts#L104)